server/routes/comments.js

```diff
@@ -3,9 +3,10 @@ import comment from '../controllers/comments';
 import loggedInUser from '../middleware/loggedInUser';
 import tokenVerification from '../middleware/tokenVerification';
 import paramCheck from '../middleware/parameterCheck';
+import Validate from '../middleware/validation';
 
 const router = express.Router();
-router.post('/commentes/:id', tokenVerification, paramCheck.parameterCheck, comment.createComment);
+router.post('/commentes/:id', tokenVerification, paramCheck.parameterCheck, Validate.addComment, comment.createComment);
```

Comment:
I think you wanted to write comments instead of commentes. Change wherever you have this word.

Reply:
Replaced